### PR TITLE
Updating mariaDB to allow the use of tolerations

### PR DESCRIPTION
**Pull request by callmeradical**

### CODE REVIEW

**Comment by viglesiasce** [comment]
cc @prydonius

File: stable/mariadb/README.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ The following tables lists the configurable parameters of the MariaDB chart and
 | `metrics.imageTag`          | Exporter image                             | `v0.10.0`                                   |
 | `metrics.imagePullPolicy`   | Exporter image pull policy                 | `IfNotPresent`                              |
 | `metrics.resources`         | Exporter resource requests/limit           | `nil`                                       |
+| `tolerations[]`             | List of node tains to tolerate			   | `[]`                                        |
```
Comment (prydonius) [inline stable/mariadb/README.md:69]: tains -> taints

File: stable/mariadb/templates/deployment.yaml [hunk 2]
```diff
@@ -10,6 +10,10 @@ metadata:
 spec:
   template:
     metadata:
+      {{- if and (.Values.tolerations) (le .Capabilities.KubeVersion.Minor "5") }}
```
Comment (prydonius) [inline stable/mariadb/templates/deployment.yaml:13]: IMO we can remove this, we only really support Kubernetes N and N-1 (1.7 and 1.8 today).

**Comment by unguiculus** [comment]
Marking stale.

**Comment by unguiculus** [comment]
Closing as stale. Feel free to reopen if/when you get a change to work on this again.

**Pull request closed without merging**